README.md

```diff
@@ -7,6 +7,56 @@ Create a community of like minded authors to foster inspiration and innovation
 by leveraging the modern web.
 
 ---
+## Application local setup
+
+### Technologies used to build the application
+1. [Django](https://www.djangoproject.com/)
+2. [Django Rest-Framework](https://www.django-rest-framework.org/)
+3. [Brew](https://brew.sh/)
```

Comment:
I think this is not used in this application

Reply:
Fixed that.